app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,100 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
```

Comment:
in js and css files, we use ' instead of "

Reply:
Done